components/GotApi.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import {
+	Text,
+	ActivityIndicator,
+	ScrollView,
+	TouchableOpacity,
+	StyleSheet,
+} from "react-native";
+import { Accelerometer } from "expo-sensors";
+
+const GotApi = () => {
+	const [quote, setQuote] = useState({});
+	const [loading, setLoading] = useState(false);
+
+	//communication between code and position of phone
+	const [subscription, setSubscription] = useState(null);
```

Comment:
What exactly did the subscription do? Is it just connected to the shaking part? Simply "subscribing" the movement part of the app and rendering?